client/my-sites/exporter/guided-transfer-options.jsx

```diff
@@ -18,7 +19,13 @@ export default React.createClass( {
 	},
 
 	purchaseGuidedTransfer() {
-		page( `/settings/export/${this.props.siteSlug}/guided` );
+		const { siteSlug } = this.props;
+		if ( config.isEnabled( 'manage/export/guided-transfer' ) ) {
+			page( `/settings/export/${ siteSlug }/guided` );
+		} else {
+			// Redirect to legacy guided transfer
+			window.location = `https://${ siteSlug }/wp-admin/paid-upgrades.php?product=40&view=purchase&source=export`;
```

Comment:
@jordwest I think we should use the site url here instead of `siteSlug` by getting it from the store and passing it in as a prop. Then we could have 

```js
window.location = siteUrl + '/wp-admin/paid-upgrades.php?product=40&view=purchase&source=export';
```